content/docs/user-guide/managing-external-data.md

```diff
@@ -112,16 +123,11 @@ Please check that you are able to connect both ways with tools like `ssh` and
 ### HDFS
 
 ```dvc
-# Add HDFS remote to be used as cache location for HDFS files
```

Comment:
should we put some summary of these comments above of after the code blocks?

Reply:
It's all under the `## Examples` header now as a numbered list 🙂

Maybe we should make all these H3s into expandable details sections so that you don't have to scroll that much between the numbered list and the actual example?